app/controllers/comments_controller.rb

```diff
@@ -13,6 +17,8 @@ def create
     end
   end
 
+  def edit; end
+
   def update
     if @comment.update(comment_params)
```

Comment:
variable comment is not assigned, we ll probably get an error for undefined method or variable for nil.